openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
```

Comment:
What about a more descriptive nickname?  I'm actually not sure what the most concise description would be.  Would `analytics-data-api-client` be ok?